.build/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Initialize flags for config.json and private key
+CONFIG_EXISTS=false
+PK_EXISTS=false
+
+# Check if config.json exists
+if [ -f /app/config.json ]; then
+  echo "config.json detected"
+  CONFIG_EXISTS=true
+else
+  echo "config.json not found"
+fi
+
+# Check if data-proxy-private-key.json exists
+if [ -f /app/data-proxy-private-key.json ]; then
+  echo "data-proxy-private-key.json detected"
+  PK_EXISTS=true
+elif [ -n "$SEDA_DATA_PROXY_PRIVATE_KEY" ]; then
+  # If private key file does not exist, check if the private key is provided via environment variable
+  echo "Private key provided via environment variable"
+  echo "$SEDA_DATA_PROXY_PRIVATE_KEY" >/app/data-proxy-private-key.json
+  PK_EXISTS=true
```

Comment:
I don't think this will work, as the key file is a JSON file with both the public and private key. I'd imagine most people would only supply the private key as an environment variable.

Reply:
If the env var is set the data-proxy uses that to construct the private/public key pair and won't even attempt to read the key file.